scripts/create_newcase

```diff
@@ -95,6 +95,10 @@ def parse_command_line(args, cimeroot):
                         help="Used to indicate that create_newcase was called from create_test"
                         "- do not use otherwise")
 
+    parser.add_argument("--config-build", action="store_true",
```

Comment:
I'm wondering if it wouldn't be better to use an environment variable to control this behavior.